src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -105,18 +106,30 @@ object DockerPlugin extends AutoPlugin {
     dockerGroupLayers := {
       val dockerBaseDirectory = (defaultLinuxInstallLocation in Docker).value
       // Ensure this doesn't break even if the JvmPlugin isn't enabled.
-      val artifacts = projectDependencyArtifacts.?.value.getOrElse(Nil).map(_.data).toSet
+      var artifacts = projectDependencyArtifacts.?.value.getOrElse(Nil).map(_.data).toSet
+
+      ClasspathJarPlugin.autoImport.packageJavaClasspathJar.?.value match {
```

Comment:
This is probably the easiest way to achieve this :+1: 

I would add a comment why the DockerPlugin cross references another plugin that is unrelated at first sight:

```suggestion
      // add the classpath jar to the project artifacts to improve layer caching as it is created by
      // the ClasspathJarPlugin and is not part of the projectDependencyArtifacts
      ClasspathJarPlugin.autoImport.packageJavaClasspathJar.?.value match {
```

Reply:
Added that comment :smile: